assembl/static2/js/app/components/cookiesBar.jsx

```diff
@@ -0,0 +1,50 @@
+// @flow
+import React from 'react';
+import { Translate } from 'react-redux-i18n';
+import { Link } from 'react-router';
+import { Button } from 'react-bootstrap';
+import classnames from 'classnames';
+import { getCookieItem, setCookieItem, getDiscussionSlug } from '../utils/globalFunctions';
+import { get } from '../utils/routeMap';
+
+type State = {
+  hide: ?boolean | string
```

Comment:
Why boolean or string ? it can lead to errors (a non empty string like `false` is truthy)
If it is because `getCookieItem('_ACCEPTED_COOKIES_')` returns a string, I guess you have to write something like : `getCookieItem('_ACCEPTED_COOKIES_') === 'true' ? true : false`